pkg_ci.es.Rmd

```diff
@@ -120,9 +120,9 @@ Para más detalles sobre este servicio, consulta [la página de ayuda de OpenCPU
 ## Aún más servicios de CI: rOpenSci docs {#rodocsci}
 
 Después de la transferencia a la organización GitHub "ropensci" de rOpenSci, cada *push* al repo de GitHub disparará la construcción (o actualización) del sitio web de tu paquete usando pkgdown. 
-Puedes encontrar el estado de este proceso en la URL `https://dev.ropensci.org/job/package_name` (por ejemplo: [para `magick`](https://dev.ropensci.org/job/magick)).
+Puedes encontrar el estado de este proceso en la URL `https://ropensci.r-universe.dev/ui#packages` y en el "commit status".
```

Comment:
se traduce "commit status"?

Reply:
Creo que traduciría "y en el estado del *commit*"

Reply:
en este caso tal vez puedo poner el enlace https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/collaborating-on-repositories-with-code-quality-features/about-status-checks aqui y en la version inglesa? O conoces algo mejor para explicar lo que es? :thinking:

Reply:
Si. Github te permite seleccionar el idioma y hace una traducción on the fly (bastante feita....pero bueno)

Reply:
ahora veo que no es de verdad una documentacion de lo que son estados de commit. ahora mismo solo encuentro cosas asi que hablan más de estados en el contexto de PRs. sigo buscando!

Reply:
https://ropensci.org/blog/2021/09/03/runiverse-docs/#how-it-works :grinning: